### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by mchalupa**
Fix getting initializer and add a new method.

### CODE REVIEW

File: src/org/sosy_lab/llvm_j/Value.java [hunk 1]
```diff
@@ -1403,6 +1410,18 @@ public long constIntGetSExtValue() {
     return LLVMLibrary.LLVMConstIntGetSExtValue(value);
   }
 
+  /** Returns the zero extended value for an integer constant value. */
+  public double constRealGetDouble() {
+    LLVMLibrary.LLVMBool precisionLoss = new LLVMLibrary.LLVMBool();
+    double ret =
+        LLVMLibrary.LLVMConstRealGetDouble(value, precisionLoss);
+
+    if (Utils.llvmBoolToJavaBool(precisionLoss))
```
Comment (lembergerth) [inline src/org/sosy_lab/llvm_j/Value.java]: To fix the Checkstyle failure:
```suggestion
    if (Utils.llvmBoolToJavaBool(precisionLoss)) {
```

File: src/org/sosy_lab/llvm_j/Value.java [hunk 2]
```diff
@@ -1403,6 +1410,18 @@ public long constIntGetSExtValue() {
...
 
+  /** Returns the zero extended value for an integer constant value. */
+  public double constRealGetDouble() {
+    LLVMLibrary.LLVMBool precisionLoss = new LLVMLibrary.LLVMBool();
+    double ret =
+        LLVMLibrary.LLVMConstRealGetDouble(value, precisionLoss);
+
+    if (Utils.llvmBoolToJavaBool(precisionLoss))
+        throw new AssertionError("Lost precision while convering float: " + value);
+
```
Comment (lembergerth) [inline src/org/sosy_lab/llvm_j/Value.java]:
```suggestion
}
```

**Comment by mchalupa** [comment]
Thanks for the review! I did the proposed changes.

**Comment by lembergerth** [comment]
Thanks!

**Pull request merged**